recipe/patches/0001-register-cmake-exe-for-win.patch

```diff
@@ -0,0 +1,51 @@
+From e3b6c5bc4fe47fdb8d1fab5b070f4e29f04b3921 Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Thu, 1 Aug 2024 20:04:24 +0200
+Subject: [PATCH] register cmake exe for win
+
+The parameters used in these tests have a lot of \ that
```

Comment:
Do you have the erros at hand? I'd like to see what the error was.

Reply:
it's this one https://github.com/scikit-build/scikit-build-core/issues/843